src/server/api/controllers/users.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+const moment = require('moment-timezone');
+
+const createUser = async (body) => {
```

Comment:
Hi. You will want to add additional validation here. Remember a city can not be number, for instance.
Validation should happen in its own function. I suggest just creating an issue for now